GuidedTutorials/MultiFab/main.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import amrex.space3d as amr
+
+# Initialize AMReX
+amr.initialize([])
+
+if amr.ParallelDescriptor.IOProcessor():
+    print(f"Hello world from pyAMReX version {amr.__version__}\n")
+
+# Goals:
+# * Define a MultiFab
+# * Fill a MultiFab with data
+# * Plot it
+
+# Parameters
+
+# Number of data components at each grid point in the MultiFab
+ncomp = 1
+# How many grid cells in each direction over the problem domain
+n_cell = 32
+# How many grid cells are allowed in each direction over each box
+max_grid_size = 16
+
+# BoxArray: abstract domain setup
+
+# Integer vector indicating the lower coordinate bounds
+dom_lo = amr.IntVect(0,0,0)
+# Integer vector indicating the upper coordinate bounds
+dom_hi = amr.IntVect(n_cell-1, n_cell-1, n_cell-1)
+# Box containing the coordinates of this domain
+domain = amr.Box(dom_lo, dom_hi)
+
+# Will contain a list of boxes describing the problem domain
+ba = amr.BoxArray(domain)
+
+# Chop the single grid into many small boxes
+ba.max_size(max_grid_size)
+
+# Distribution Mapping
+dm = amr.DistributionMapping(ba)
+
+# Define MultiFab
+mf = amr.MultiFab(ba, dm, ncomp, 0)
+mf.set_val(0.)
+
+# Geometry: physical properties for data on our domain
+real_box = amr.RealBox([0., 0., 0.], [1. , 1., 1.])
+
+coord = 0  # Cartesian
+is_per = [0, 0, 0] # periodicity
+geom = amr.Geometry(domain, real_box, coord, is_per)
+
+# Calculate cell sizes
+dx = geom.data().CellSize() # dx[0]=dx dx[1]=dy dx[2]=dz
+
+# Fill a MultiFab with data
+mf_val = 0.
+for mfi in mf:
+    bx = mfi.validbox()
+    # Preferred way to fill array using NumPy operations:
+    # - mf_array is indexed in reversed order (n,z,y,x)
+    # - indices are local (range from 0 to box size)
+    mf_array = np.array(mf.array(mfi), copy=False)
```

Comment:
That way it will be the same index order as AMReX (Fortran):
```suggestion
    mf_array = np.array(mf.array(mfi), copy=False).T
```

Reply:
Alternatively, in the future, we can rely on these helpers:
https://github.com/AMReX-Codes/pyamrex/pull/88